tests/test_api.py

```diff
@@ -473,3 +473,73 @@ def test_unenrolls_api(self):
             data['population']['variant'][1],
             {'window': self.window2[0].isoformat(), 'count': 10}
         )
+
+
+class TestEnrollmentCountsApi(EnrollmentBaseTestCase):
+
+    def setUp(self):
+        self.url = reverse('v2-experiment-enrolls', args=['pref-flip-1'])
+        self.create_data()
+
+    def test_enrolls_404(self):
+        response = self.client.get(
+            reverse('v2-experiment-enrolls', args=['foo']))
+        self.assertEqual(response.status_code, 404)
+
+    def test_enrolls_api(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        data = response.json()
+        result_date1 = datetime.datetime(self.window1[0].year,
+                                        self.window1[0].month,
+                                        self.window1[0].day,
+                                        self.window1[0].hour,
+                                        0, 0, 0).isoformat()
+        result_date2 = datetime.datetime(self.window2[0].year,
+                                        self.window2[0].month,
+                                        self.window2[0].day,
+                                        self.window2[0].hour,
+                                        0, 0, 0).isoformat()
```

Comment:
Perhaps a slightly easier way to simulate the `TruncHour` would be to use `.replace` on the datetime object, like this:
`result_date1 = self.window[1].replace(minute=0, second=0)`. And then you don't need the string concatenation of the timezone bit.

Reply:
I tried that but there was no way to floor the microseconds since that only allows replacement up to seconds.

Reply:
This works... `self.window[1]..replace(minute=0, second=0, microsecond=0).isoformat()`